feat: How to Delete an account (#149)
citynetwork/docs: docs/howto/account-billing/Delete-account.md

# Delete account

If you unfortunately need to cancel or leave us for any reason, you can delete your account in our [{{gui}}](https://{{gui_domain}}).
Log in and click on the Profile button at the top right.

![Orange avatar person with white
background](assets/picture-profile.png)

You will now see the _Account settings_ page. Choose the _Settings_ tab, in which you can delete your account.

![A Settings tab with a red box around it](assets/accountsettings.png)

Scroll to the bottom until you see the image below and if you absolutely sure click on the red button _Delete Account_ and follow the procedure.

![Black text on white background with info what will happen when you click on the delete button](assets/Delete-account-picture.png)

> Your services will be shut off in 3 days and from now your services stop costing you money.

Reviewer: Assuming all user assets will be wiped in 72 hours, I would write this as follows:

```suggestion
Any assets you may still have in {{brand}} will be wiped after 72 hours. Please keep in mind that during that time you will be charged as usual.
```

Author: No, the assets will only be suspended after 3 days. After 10 days we will start deleting servers and other stuff.

Shall I instead write 

Any assets you may still have in {{brand}} will be wiped after 10 days. Please keep in mind that during that time you will be charged as usual.

Reviewer: Aha, so assets get wiped irrevocably after 10 and not after 3 days. Does this make any difference in the pricing structure? Is the customer getting charged as usual during this 10-day time period? If so, then I would suggest this change:

 ```suggestion
Any assets you may still have in {{brand}} will be suspended after 3 days, and wiped irrevocably 7 days after suspension. Please be aware that during these 10 days, you will be getting charged as usual.
```

> If you regret what you just did, please contact us directly at support@cleura.com and we will do everything in our power to save your account.

Reviewer 2: These sound awkward and will need rephrasing. I'll leave it to @colder-is-better to provide suggestions. :)

Author: It´s me who wanna save any customer at any price :-)

Author: But colder-is-better wrote it better

> If you regret what you just did, please contact us directly at support@cleura.com and we will do everything in our power to save your account.

Author: Sound better - i fix

> If you regret what you just did, please contact us directly at support@cleura.com and we will do everything in our power to save your account.

Author: You want me to add red color on the words **If** and **and** ?

> If you regret what you just did, please contact us directly at support@cleura.com and we will do everything in our power to save your account.

Author: If wee choose 10 days should maybe the last text be:

If you change your mind during the first 3 days, contact us by emailing support@cleura.com and we will **not delete** your account.

> If you regret what you just did, please contact us directly at support@cleura.com and we will do everything in our power to save your account.

Reviewer: OK, so even though assets are irrevocably wiped after 10 days, the customer has only 3 days to change their mind. If this is the case indeed, then taking into account Florian's suggestion for contacting support, I would suggest this change:

```suggestion
If you change your mind during the first 3 days please [contact support](https://servicecenter.citynetwork.eu/servicedesk), and we will **not delete** any of your assets, nor your account.```

# Delete account

If you unfortunately need to cancel or leave us for any reason, you can delete your account in our [{{gui}}](https://{{gui_domain}}).
Log in and click on the Profile button at the top right.

![Orange avatar person with white
background.](assets/picture-profile.png)

You will now see the _Account settings_ page. Choose the _Settings_ tab, in which you can delete your account.

![The "Customer Info" tab with a green "Update" button to complete the
changes](assets/picture-changecustomerinfo2.png)

> Scroll to the bottom until you see the image below and if you absolutely sure click on the red button _Delete Account_ and follow the procedure.

Author: Of course, my fault :-)

# Deleting your account

> If you unfortunately need to cancel or leave us for any reason, you can delete your account in our [{{gui}}](https://{{gui_domain}}).

Reviewer: ```suggestion
In case you decide to close your account in {{brand}}, you may do so via the [{{gui}}](https://{{gui_domain}}).
```

Log in and click on the Profile button at the top right.

![Orange avatar person with white background.](assets/picture-profile.png)

> You will now see the _Account settings_ page. Choose the _Settings_ tab, in which you can delete your account.

Reviewer: ```suggestion
A pane named _Account settings_ will slide over from the right-hand side of the browser window. To delete your account, click the _Settings_ tab.
```

![A Settings tab with a red box around it](assets/accountsettings.png)

> Scroll to the bottom until you see the image below and if you are absolutely sure, click on the red button _Delete Account_.

Reviewer: ```suggestion
Scroll to the bottom of the pane, and click the red button named _Delete Account_.
```

![Black text on white background with info what will happen when you click on the delete button](assets/Delete-account-picture.png)

> Choose a reason for leaving and type in your current password. Click on red button _Delete Account_. Now your account has been requested to be deleted.

Reviewer: ```suggestion
Choose a reason for leaving and type in your current password. Click the red button named _Delete Account_. That will create a request for account deletion.
```

# Deleting your account

> In case you decide to close your account in {{brand}}, you may do so via the [{{gui}}](https://{{gui_domain}}).

Reviewer 2: ```suggestion
In case you decide to close your {{brand}} account, you may do so via the [{{gui}}](https://{{gui_domain}}).
```

Log into your account and click on the profile button at the top right.

![Orange avatar person with white background.](assets/picture-profile.png)

A pane named _Account settings_ will slide over from the right-hand side of the browser window. To delete your account, click the _Settings_ tab.

![A Settings tab with a red box around it](assets/accountsettings.png)

Scroll to the bottom of the pane, and click the red button named _Delete Account_.

![Black text on white background with info what will happen when you click on the delete button](assets/Delete-account-picture.png)

> Choose a reason for leaving and type in your current password. Click the red button named _Delete Account_. That will create a request for account deletion.

Reviewer: Since this is the second time we click on the red button, I suggest the following _slight_ change:

```suggestion
Choose a reason for leaving and type in your current password. Click the red button named _Delete Account_ one more time. That will create a request for account deletion.
```

![Picture where you choose a option why you leave us. A field where you type your current password with a red delete button.](assets/reasonforleaving.png)

Three (3) days after sending the request for deletion, any running virtual machines will be shut off. Then, after ten (10) days, all virtual machines and any other resources connected to your account will be deleted, **including** backups.

After the last invoice has been paid, we will delete your account. This entire process may take up to thirty (30) days after the last invoice is paid.

> If you change your mind during the first 3 days please contact [Service Center](https://servicecenter.citynetwork.eu/servicedesk), and we will **not delete** any of your assets, nor your account.

Reviewer: To be consistent with how we specify the time (in days), I'm now suggesting this:

```suggestion
If you change your mind during the first three (3) days, please get in touch with [Service Center](https://servicecenter.citynetwork.eu/servicedesk), and we will **not delete** your assets or your account.
```

I also went with "get in touch", for I find it friendlier than "contact" :)

> If you change your mind during the first three (3) days, please get in touch with [Service Center](https://servicecenter.citynetwork.eu/servicedesk), and we will **not delete** your assets or your account.

Reviewer 2: ```suggestion
If you change your mind during the first three (3) days, please get in touch with our [{{support}}](https://{{support_domain}}/servicedesk), and we will **not delete** your assets or your account.
```
